packages/storybook-test/stories/data-badge/data-badge.stories.tsx

```diff
@@ -39,7 +44,22 @@ type Story = StoryObj<typeof meta>;
 export const Default: Story = {
   name: 'Data Badge',
   args: {
-    children: '42',
+    children: 'Bevoegd gezag: Gemeente',
+  },
+  decorators: [
+    (Story) => (
+      <>
+        <Heading level={1}>Zoekresultaat voor "zonnepaneel"</Heading>
+        <Paragraph>Geselecteerde filters: {Story()}</Paragraph>
```

Comment:
Denk netter om overal gewoon dit te doen en niet de ene keer `{Story()}` en de andere keer `<Story />` te gebruiken.

```suggestion
        <Paragraph>Geselecteerde filters: <Story /></Paragraph>
```